codalab/apps/web/templates/web/competitions/edit.html

```diff
@@ -37,6 +37,38 @@
                             {% endif %}
                         </div>
                     </div>
+                {% elif field.label == 'Admins' %}
+                    <style>
```

Comment:
let's move this into one of the base CSS files

Reply:
I believe you should check out `imports.less` for how it works, maybe you'll want to add `select2_custom.less` and import it there? may not work how I remember...

Reply:
Also, sorry for spam, you'll want to do a `docker-compose up -d --build` to make it re-do the LESS =\